bin/satis

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env php
 <?php
-
+ini_set ("memory_limit","-1");
+error_reporting(E_ERROR | E_WARNING | E_PARSE);
+//报告所有错误
+error_reporting(E_ALL);
+ini_set("display_errors","On");
```

Comment:
Does this belong into your pull request? Non-englisch comment, increasing memory limit, double setting the error reporting, switching on error display all looks pretty much like leftovers from debugging.